src/openloco/openloco.cpp

```diff
@@ -561,6 +563,74 @@ namespace openloco
         }
     }
 
+    static void sub_496A84(int32_t edx)
+    {
+        // This is responsible for updating the snow line
+        registers regs;
+        regs.edx = edx;
+        call(0x00496A84, regs);
+    }
+
+    // 0x004968C7
+    static void date_tick()
+    {
+        if ((addr<0x00525E28, uint32_t>() & 1) && !is_editor_mode())
```

Comment:
Return early instead of wrapping the entire function in an `if` block?

Reply:
This isn't how I code any more, I prefer single entry, single exit.